app/frontend/courses/report/components/CoursesReport__Overview.css

```diff
@@ -120,3 +120,7 @@
   fill: theme("colors.orange.200");
   stroke: theme("colors.orange.500");
 }
+
+.word-break {
+  word-break: break-all;
```

Comment:
Do we need `word-wrap: break-word` here?

If not, then the `break-all` class is already present in tailwindCSS.

Reply:
> Do we need word-wrap: break-word here

No, it is working fine without above.

Updated to use tailwind class